src/sass/components/_color-settings.scss

```diff
@@ -0,0 +1,26 @@
+@use '../config';
+
+[data-setting] {
+  align-items: center;
+  column-gap: var(--gutter);
+  display: grid;
+  grid-template:
+    'format-label' auto
+    'format-input' auto / 1fr;
+  justify-content: end;
+
+  @include config.above('sm-page-break') {
+    --align-label: right;
+
+    grid-template: 'format-label format-input' auto / 1fr auto;
+  }
+
+  label {
+    grid-area: format-label;
+    text-align: var(--align-label, left);
```

Comment:
I can't see a difference when I change this. Do we still need it?

Reply:
oh, probably not - good catch.

Reply:
@dvdherron fixed this, and improved the small screen layout